### ISSUE TITLE
Uncaught TypeError: Cannot read property 'isHorizontal' of undefined at Swiper.getTranslate

### ISSUE THREAD

**Issue by anvucodespot** [nolimits4web/swiper#2585]
Hello,

I use the package from npm, I think i have the latest version. After setup a slider, everything seem be good. But when I swipe the slider, this error be alerted. How can I fix it?

Uncaught TypeError: Cannot read property 'isHorizontal' of undefined
    at Swiper.getTranslate (swiper.esm.bundle.js?41d6:910)
    at Swiper.loopFix (swiper.esm.bundle.js?41d6:1333)
    at swiper.autoplay.timeout.Utils.nextTick (swiper.esm.bundle.js?41d6:5939)

Thank you for helping.

**Comment by danfri86** [comment]
I'm experiencing the same error, but with autoplay: on and loop: true. The error is happening when the slide is supposed to switch automatically.
Did you find a solution?

If I set loop: false it works.

**Comment by david-treblig** [comment]
I have a similar error and I use the built dist file, so missing build files shouldn't be an issue in my case. I have the problem only while looping. Line 934 throws an error because it tries ``` win.getComputedStyle(el, null)``` on an undefined element since in function  ```loopFix``` it calls ```getTranslate()``` without parameters.

**Comment by nolimits4web** [comment]
@david-treblig 

> I have a similar error and I use the built dist file

`esm` files in dist folder are also must be transpiled. If you use transpiled umd version then please provide a JSFiddle or live example with the issue

**Comment by david-treblig** [comment]
I think my issue is different, now that I've looked at it further. I have this on a live website right now : http://patrimoinecotedebeaupre.com. The rotating images in the header are a Swiper instance as well, but the one causing the issue is the one down where it says "Type architecturaux". It only does this when I activate "loop" which I had disabled untill I'd find an answer, but I reactivated it for demonstration purposes. 

Both the header swiper and this one are initialized on load. The problematic one is initialized like this : 

```js
new Swiper('.block-tax-slider', {
    direction: 'horizontal',
    loop: true,
    slidesPerView: 4,
    spaceBetween: 46,
    speed: 750,
    autoplay: {
        delay: 7000,
        disableOnInteraction: false
    },
    navigation: {
        nextEl: '.tax-slider-button-next',
        prevEl: '.tax-slider-button-prev',
    },
    breakpoints: {
        768: {
            slidesPerView: 1,
        },
        1280: {
            slidesPerView: 2,
        },
        1440: {
            slidesPerView: 3,
        }
    }
});
```
The error triggers once on autoplay navigation, on every click prev / next navigation and on every single movement of the drag or swipe. I was wondering if placing my nav elements outside the main container was the issue, but it doesn't seem to be the case. I've tried to replicate the issue in a JS Fiddle with the same init values and it won't reproduce... -_-x  (https://jsfiddle.net/w2pmhqed/18/). I'm really scratching my head over this one!

The fact is that nothing is broken... everything works as expected. It's just reporting that the $wrapperEl is not an instance of Element once every two calls. It's happening in the loopFix function on swiper.getTranslate()...
